src/content/docs/concepts/programs.mdx

```diff
@@ -41,3 +41,17 @@ A Membrane program contains at least three files:
 3. **`memconfig.lock`:** tracks versions of nodes in your graph. It's also automatically generated, and you should never have to edit it manually
 
 Your program's directory may also include a `README.md`, `package.json` with npm dependencies, `.git`, and whatever other helpers and structure you'd like.
+
+## Stopping and deleting programs
+
+To stop a running program, right click the program in the Membrane Explorer and select `Kill` from the dropdown menu. A program must be stopped in order to delete it.
+
+:::caution
+When you stop a running program, it's `state` will be lost.
+:::
+
+To delete a program, right click the program's folder in your file explorer and select `Delete` from the dropdown menu. Or, open the command palette (`Cmd+Shift+P` on Mac) and type _Membrane: Delete Program_. A message will appear in the bottom bar of your editor upon deletion.
+
+:::danger
+Deleting a program is a permanent action.
```

Comment:
Is this true? I think no because we soft delete?

I was putting myself in the shoes of someone reading it who might think "ugh my programs should just be safely checked into github then instead of your cloud".

Reply:
For all intents and purposes it _is_ a permanent action. We don't give users a way to restore their programs and I don't think we should indicate otherwise. One option a user can take is to right click a programs source directory and `Download`. That'll let them download it to their local file system. It's a little finicky because it seems to actually use the web fs so it has some special rules around how it can be used, but that's always an option.